front-end/src/App.js

```diff
@@ -1,9 +1,227 @@
-import React from 'react'
+import React, { useState } from "react";
+import "./App.css";
+import Navbar from "./components/navbar"
+import Footer from "./components/footer"
+import comicCity from "./extras/Comic City.png";
+import marsRed from "./extras/marsRed.png";
+import blueMoon from "./extras/vampireBlue.png";
+import mushoku from "./extras/mushokuTensei.png";
+import noblesse from "./extras/Noblesse.png";
+import TianGuanCiFu from "./extras/TianGuanCiFu.png"
+import NanoMachine from "./extras/NanoMachine.png"
+import SwordSage from "./extras/SowrdSage.png"
+import SwordKing from "./extras/SwordKing.png"
+import footerLogo from "./extras/ComicCityFooter.png"
 
-const App = () => {
+
+
+// import Swiper core and required modules
+import {EffectCoverflow, Navigation, Pagination,Autoplay, Scrollbar, A11y } from 'swiper';
+
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+// Import Swiper styles
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+import 'swiper/css/scrollbar';
+import 'swiper/css/bundle';
+import 'swiper/css/effect-coverflow'
+
+function App() {
+  
+  return (
+    <>
+
+      <Navbar />
+      <div className="App">
+        <div id="navbar">
+        
+          <div id="logoContainer">
+          </div>
+        </div>
+
+        <div><Carousel /></div>
+        <div><PopularComics /></div>      
+        <div><LastRead /></div>
+
+      </div>
+
+      <footer><Footer /></footer>
```

Comment:
No need to add footer tag here, without it you can add <Footer/> component.

Reply:
Okay :)